pylint_nautobot/utils.py

```diff
@@ -130,14 +130,17 @@ def get_model_name(node: ClassDef) -> str:
     return get_model_name_from_attr(model_attr)
 
 
-def find_model_name_from_queryset(node: NodeNG) -> str:
+def get_model_name_from_queryset(node: NodeNG) -> str:
     """Get the model name from the queryset assignment value."""
     while node:
         if isinstance(node, Call):
             node = node.func
         elif isinstance(node, Attribute):
-            if node.attrname == "objects" and isinstance(node.expr, Name):
-                return node.expr.name
+            if node.attrname == "objects":
+                if isinstance(node.expr, Name):
+                    return node.expr.name
+                if isinstance(node.expr, Attribute):
+                    return node.expr.attrname
```

Comment:
Can you add a short comment that explains why these two are needed and what the difference is`?

Reply:
Comments added